src/graphql/getters/getSeasonBadges/getSeasonBadges.ts

```diff
@@ -43,20 +43,20 @@ export async function getSeasonBadges({
     }
 
     const badges = data.seasonUser.badges;
-    const response = SEASON_BADGE_VARIANTS[seasonId].map((badgeVariant) => {
+    return SEASON_BADGE_VARIANTS[seasonId].map((badgeVariant) => {
       const badge = badges.find((b) => BADGE_TYPE_BADGE_VARIANT_MAP[b.badgeType] === badgeVariant);
       if (!badge) {
         return {
           variant: badgeVariant as BadgeVariant,
         };
       }
       return {
+        badgeResponseRaw: badge,
```

Comment:
is this necessary if we already have the processed data?

Reply:
yes I need this so that when I consume the SDK i can provide it with the data from the graphql